plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -1352,7 +1352,7 @@ private function do_bulk_action_remove_personal_data( $order_ids ): int {
 			}
 
 			do_action( 'woocommerce_remove_order_personal_data', $order ); // phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
-			$changed++;
+			++$changed;
```

Comment:
Let's revert those then

```suggestion
			$changed++;
```

Reply:
A linter would fix them again the next time thou?